spec/requestStubSpec.js

```diff
@@ -101,4 +101,33 @@ describe('RequestStub', function() {
       expect(stub.status).toBe(500);
     });
   });
+
+  describe('when streaming', function() {
+    it('isStream', function() {
+      var stub = new this.RequestStub('/foo');
+      stub.andReturnStream();
+
+      expect(stub.isStream()).toBe(true);
+    });
+
+    it('thrown an error if finish request and then try to cancel', function() {
```

Comment:
These errors should be thrown regardless of whether the user is using the stubbing functionality or the after the fact functionality. They're only tested here and aren't thrown from the `fakeRequest`.

Reply:
Yeah, you're right! Thanks! I have added throwing error and tests for it in https://github.com/jasmine/jasmine-ajax/pull/183/commits/7b23ae0807085f01504f08152a8aa63b51d45036